src/utils/caipConversion.ts

```diff
@@ -12,14 +15,14 @@ export enum CaipNamespace {
 }
 
 export const BitcoinCaipId = {
-  [ChainId.BITCOIN]: `${CaipNamespace.BIP122}:000000000019d6689c085ae165831e93`,
-  [ChainId.BITCOIN_TESTNET]: `${CaipNamespace.BIP122}:000000000933ea01ad0ee984209779ba`,
+  [ChainId.BITCOIN]: BitcoinCaip2ChainId.MAINNET,
```

Comment:
Switched to `@avalabs/core-chains-sdk` since that's much lighter than the Wallets SDK (and it would be included in the inpage script).